packages/styles/README.md

```diff
@@ -1,3 +1,90 @@
 # mantle:styles
 
-This package defines styles for mantle. 
\ No newline at end of file
+This package defines styles for mantle.
+
+##Style Guide
+- **Framework**: [Bootstrap](http://getbootstrap.com/css/)
+- **Icon Font**: [Font Awesome](http://fontawesome.io/)
+- **Media Queries**: [Rupture](http://jenius.github.io/rupture/)
+- **Mixin Library**: [Nib](http://tj.github.io/nib/)
+
+###Adding Files and styles
+- Create a file with the name of associated view or package.  Import files into *main.styl* **and** add them to package.js:
+  - Filename: *viewname.import.styl*
+  - Importing: `@import viewname.import`
+  - Add to package.js: `api.addFiles('viewname.import.styl');`
+- Custom mixins should be added to *mixins.import.styl*
+- Global styles should be added to *globals.inport.styl*
+- Add global variables to *varaibles.import.styl*
+
+####Syntax Patterns
+- Do not use brackets, colons, or semicolons
+- Class names should be lowercase, hyphenated and as short as possible while accurately indicating purpose
+- Try to use classes as selectors only
+- Use single quotes
+- Prefix variable names and functions with `$`
+- Browser prefixes are added with Nib and Autoprefixer
```

Comment:
Yeah we should probably add that. Its mentioned in the resources but important enough to mention in our guide.